### ISSUE TITLE
bug in traverse function

### ISSUE THREAD

**Issue by loustler** [loustler/my-setting#2]
Found bug when to use traverse function.

This function print incorrect message when you gave to command which has another option or including empty space through first parameter.

Below message is example for incorrect message.

```
[command which you gave] not found. Check this function if exist.
```

Incorrect logic and message.

Needs hot fix for this bug.

---------------------------------

traverse function을 사용할 때 버그를 발견

첫 번째 파라미터에 넘겨주는 커맨드에 빈 공간이 있거나 다른 옵션이 있으면 잘못된 메시지가 출력됨

아래의 메시지는 잘못된 메시지의 예제임.

```
[command which you gave] not found. Check this function if exist.
```

정확하지 않은 논리와 메시지임

이 버그의 빠른 수정이 필요함

**Comment by loustler** [comment]
The logic find invalid command, to use `which` command.

Below code is incorrect logic to find invalid command.

```
if [[ "$(which $MY_CMD)" == *"not found" ]]
  then
    echo "$MY_CMD not found. Check this function if exist."
    return 1
fi
```

Above logic execute some command.

Check message if contains `not found`.

If message contains `not found` then print message and finish function.

**Comment by loustler** [comment]
Fix bug in b4407b11351cdb2823f2f6289718ebef6c873252.

1. check command to use split string
